eslint.config

```diff
@@ -2,5 +2,11 @@
   "env": {
     "browser": true
   },
-  "extends": ["airbnb"]
+  "extends": ["airbnb"],
+  "rules": {
```

Comment:
I don't think `indent`, `no-use-before-define`, and `no-console` should be disabled. 

`indent` will make our overall code look consistent (which is a best practice). Enabling this will also let us automatically indent with a single command. From the [official documentation](https://eslint.org/docs/rules/indent#enforce-consistent-indentation-indent):
> The --fix option on the command line can automatically fix some of the problems reported by this rule.

`no-use-before-define` should also be enabled. Although the code won't break, I think code readability is important. For someone unfamiliar with our codebase, this might make things confusing.

As for `no-console`, the [official documentation](https://eslint.org/docs/rules/no-console#disallow-the-use-of-console-no-console) sums it up pretty well:
> In JavaScript that is designed to be executed in the browser, it’s considered a best practice to avoid using methods on console. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using console should be stripped before being pushed to production.

Reply:
Good points! I'll do a styling refactor to all of our codes as some of the files written and merged before this PR don't adhere to this. Will include it in this PR. Thanks!